firmware/projects/LVController/README.md

```diff
@@ -0,0 +1,9 @@
+# Low Voltage Controller
+
+Refer to macformula/lvcontroller
+
+the following regex can be used to extract non-empty regions of USER code in cubemx generated files.
+
+```regex
+/\* USER CODE BEGIN ([\w]*) \*/\n[\S\s]*?[\w]+[\S\s]*?/\* USER CODE END \1 \*/
```

Comment:
Curious what the use might be here given we dont push the generated code to source control?

Reply:
This was for analyzing last years code, will be removed